spec/services/multi_tenant_service_spec.rb

```diff
@@ -88,6 +88,33 @@
         expect(described_class.new(:gyr).filing_years(fake_time)).to eq [2024, 2023, 2022, 2021]
       end
     end
+
+    context "GYR 2025 after tax deadline before end of in progress intake" do
```

Comment:
Could you add one more test which makes sure that we no longer see 2020 after the "end of in-progress intake" date?

Reply:
Great call